src/app/common/stores/applications/SsvWeb/Operator.store.ts

```diff
@@ -38,7 +38,7 @@ export interface IOperator {
   dkg_address?: string,
   mev_relays?: string,
   autoSelected?: boolean
-  validatorsCount?: number,
+  validators_count: number,
```

Comment:
I need to investigate this part of code, because from api it's came as snake case, but I think when we create operator for example we send as camelCase, I will create ticket to refactor all places where we have snake case and convert it into camel case